.nancy-ignore

```diff
@@ -0,0 +1,16 @@
+# The go-libp2p dependency is not used directly in this plugin so the following
```

Comment:
i'm not sure if we're even using libp2p for ocr anymore. is this copied from CL repo?

Reply:
No. I made this comment to indicate that this repo only really uses libocr for types and interfaces and doesn't use libp2p directly. The libocr repo may still have this as a dependency even if the chainlink repo uses a different implementation. OCR2 allows for an arbitrary networking layer as it defines an expected interface, not a distinct type.